src/swtpm/swtpm.c

```diff
@@ -249,6 +250,7 @@ int swtpm_main(int argc, char **argv, const char *prgname, const char *iface)
         {"fd"        , required_argument, 0, 'f'},
         {"server"    , required_argument, 0, 'c'},
         {"runas"     , required_argument, 0, 'r'},
+        {"chroot"    , required_argument, 0, 'R'},
```

Comment:
You have to add this also to the help screens.

Reply:
OK, will do. I'll also add a test case.

Reply:
If you need a pair of available TCP ports for the test cases, please use 65468/65469

Reply:
Updated as suggested.